library/ssl_cookie.c

```diff
@@ -167,7 +167,9 @@ int mbedtls_ssl_cookie_write( void *p_ctx,
     if( (size_t)( end - *p ) < COOKIE_LEN )
         return( MBEDTLS_ERR_SSL_BUFFER_TOO_SMALL );
 
-#if defined(MBEDTLS_HAVE_TIME)
+#if defined(FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION)
+    t = 0x5af2a056;
```

Comment:
Would it be possible to achieve the same effect by setting `mbedtls_time` to a function that returns a constant in `platform_time.h`?

Reply:
Your idea looks better indeed.
What is the best way to uncomment `MBEDTLS_PLATFORM_TIME_ALT` in config.h ?

Reply:
Normally, run `scripts/config.pl set MBEDTLS_PLATFORM_TIME_ALT` before building. This modifies `config.h` in place.

We could set things up so that `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION` automatically turns on `MBEDTLS_PLATFORM_TIME_ALT`. But we don't normally have one option automatically enable another option. On the gripping hand, fuzzing mode is somewhat special, so while I'd prefer not to automatically turn on another option, in this case it wouldn't be a blocker.

Reply:
I suggest to keep it this way and document this...